ramls/pick-slips-response.json

```diff
@@ -0,0 +1,89 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Collection of items",
+  "description": "Collection of items",
+  "type": "object",
+  "properties": {
+    "items": {
```

Comment:
Why is the records that are returned from a question about pick slips called `items` rather than `pickSlips`?

Reply:
Good point. Fixed.